docs/network-flow-visibility.md

```diff
@@ -357,9 +357,17 @@ different Nodes can be preserved.
 
 If you would like to quickly try Network Flow Visibility feature, you can deploy
 Antrea, the Flow Aggregator Service and the ELK Flow Collector on the
-[Vagrant setup](../test/e2e/README.md). However, the ELK Flow Collector deployment
-requires the Vagrant Nodes to have higher memory than default, so we have to provision
-the Nodes with the `--large` option. You can use the following command:
+[Vagrant setup](../test/e2e/README.md). You can use the following command:
+
+```shell
+./infra/vagrant/provision.sh
+./infra/vagrant/push_antrea.sh --flow-collector ELK
+```
+
+If you would like to try workloads with high resource requirements, you can change the
```

Comment:
I did some more tests on the flow rate and noticed my previous observation on the flow record ingestion rate is related to the logstash instead of the ELK. In my tests, the flow aggregator can record all the flow in the cluster, but some of them are lost in the logstash logs, while the number of records in Kibana is the same as the one in logstash.

Reply:
The reason I observed a different flow record ingestion rate is that the logstash may lose more the testing flows I triggered in the pod in about the first 20 minutes from the initialization. After that, the flow record ingestion rate will increase regardless of the ELK memory.

Reply:
Ok. We seem to hit the logstash bottleneck first rather than elastic search bottleneck when high ingestion rate is there, which is in line with the scale tests.

Let's reword this description to be more direct rather than workloads which is somewhat vague IMO:

"If you would like to deploy elastic search with high resources,you can ..."

Reply:
Sounds good. I have updated the description.